ArduPlane/parachute.cpp

```diff
@@ -56,3 +59,55 @@ bool Plane::parachute_manual_release()
 
     return true;
 }
+
+// Code to detect loss of control for ArduPlane
+#define PARACHUTE_CHECK_TRIGGER_SEC         1  // 1 second of loss of control triggers the parachute
```

Comment:
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  dupuyyann@gmail.com
    Domain zyapps.com has exceeded the max emails per hour (454/400 (113%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from o3.sgmail.github.com ([192.254.112.98]:36272)
	by serveur.ex2-p10.com with esmtps (TLSv1.2:ECDHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.86_1)
	(envelope-from <bounces+848413-8f06-yann.dupuy=zyapps.com@sgmail.github.com>)
	id 1afmao-000dkX-52
	for yann.dupuy@zyapps.com; Tue, 15 Mar 2016 11:52:58 +0100
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed; d=github.com; 
	h=from:reply-to:to:in-reply-to:references:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=/b6sBdMZq6So4vxRhMaVo4dsq/8=; b=MjTDwnIWNgfgRXIl
	nh/9kloCfSiQjqEWZCTY3PpwFs0vjbHwHyJc0sfu6twocqBxgWATD5mqJGm3H2wh
	SMMV9Wv2WubDqpiYV0byC1CMra3O4GZgl3t06AIJHxGelp8aQA5wOvBQRLQtEvaF
	499ZD/wdlMMNWjsia00DnyzpsrY=
Received: by filter0912p1mdw1.sendgrid.net with SMTP id filter0912p1mdw1.15600.56E7E95452
        2016-03-15 10:52:04.880027198 +0000 UTC
Received: from github-smtp2a-ext-cp1-prd.iad.github.net (github-smtp2a-ext-cp1-prd.iad.github.net [192.30.253.16])
	by ismtpd0001p1iad1.sendgrid.net (SG) with ESMTP id CPLwXusXRBW6ilvwUf0VsA
	for <yann.dupuy@zyapps.com>; Tue, 15 Mar 2016 10:52:04.782 +0000 (UTC)
Date: Tue, 15 Mar 2016 03:52:04 -0700
From: Rimvydas Naktinis <notifications@github.com>
Reply-To: diydrones/ardupilot <reply@reply.github.com>
To: diydrones/ardupilot <ardupilot@noreply.github.com>
Message-ID: <diydrones/ardupilot/pull/3703/r56144970@github.com>
In-Reply-To: <diydrones/ardupilot/pull/3703@github.com>
References: <diydrones/ardupilot/pull/3703@github.com>
Subject: Re: [ardupilot] Plane: basic automatic emergency parachute release at
 critical altitude error in AUTO. (#3703)
Mime-Version: 1.0
Content-Type: multipart/alternative;
 boundary="--==_mimepart_56e7e9549f2f3_5c9a3fba4a0e12a01564035";
 charset=UTF-8
Content-Transfer-Encoding: 7bit
Precedence: list
X-GitHub-Sender: naktinis
X-GitHub-Recipient: yanndupuy
List-ID: diydrones/ardupilot <ardupilot.diydrones.github.com>
List-Archive: https://github.com/diydrones/ardupilot
List-Post: <mailto:reply@reply.github.com>
List-Unsubscribe: <mailto:unsub+000df0dcc75a8087ef782feaa6fadc530f6ba412ab11ea4e92cf0000000112ffab5492a170ce0358b44a@reply.github.com>,
 <https://github.com/notifications/unsubscribe/AA3w3Eu3IKrqr-1AoM-W3VkwVosm571Rks5pto9UgaJpZM4HouR4>
X-Auto-Response-Suppress: All
X-GitHub-Recipient-Address: yann.dupuy@zyapps.com
X-SG-EID: /bDRkhSo/GH2gzZXMo5AKhbuTgLDPkexBjh/BYa140EGDZ0uXWBZO0LMe0b06Or59s7ROZux/f7I+n
 6NxlOK2ULC9IpcBugC75z+KoVX4L9zIyj+da0CCR7uVf8jx44sxaEpeOpeLTn+Rfs0/gcoO7Flkk5H
 mvuu9RFu9uTvxj5LbTtzi0wUYLLxGD7IZEAcl/W1/W6AyWv/F48BkEfz0w/y3GWE2V6f91GoqIKvAX
 M=
X-Spam-Status: No, score=-1.4
X-Spam-Score: -13
X-Spam-Bar: -
X-Ham-Report: Spam detection software, running on the system "serveur.ex2-p10.com",
 has NOT identified this incoming email as spam.  The original
 message has been attached to this so you can view it or label
 similar future email.  If you have any questions, see
 root\@localhost for details.
 
 Content preview:  > @@ -56,3 +59,55 @@ bool Plane::parachute_manual_release()
    > > return true; > } > + > +// Code to detect loss of control for ArduPlane
    > +#define PARACHUTE_CHECK_TRIGGER_SEC 1 // 1 second of loss of control triggers
    the parachute [...] 
 
 Content analysis details:   (-1.4 points, 5.0 required)
 
  pts rule name              description
 ---- ---------------------- --------------------------------------------------
 -2.3 RCVD_IN_DNSWL_MED      RBL: Sender listed at http://www.dnswl.org/, medium
                              trust
                             [192.254.112.98 listed in list.dnswl.org]
 -0.0 RCVD_IN_MSPIKE_H3      RBL: Good reputation (+3)
                             [192.254.112.98 listed in wl.mailspike.net]
 -0.0 RP_MATCHES_RCVD        Envelope sender domain matches handover relay domain
  0.0 HTML_MESSAGE           BODY: HTML included in message
  1.0 HTML_IMAGE_ONLY_16     BODY: HTML: images with 1200-1600 bytes of words
 -0.1 DKIM_VALID_AU          Message has a valid DKIM or DK signature from author's
                             domain
  0.1 DKIM_SIGNED            Message has a DKIM or DK signature, not necessarily valid
 -0.1 DKIM_VALID             Message has at least one valid DKIM or DK signature
 -0.0 RCVD_IN_MSPIKE_WL      Mailspike good senders
X-Spam-Flag: NO

----==_mimepart_56e7e9549f2f3_5c9a3fba4a0e12a01564035
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

> @@ -56,3 +59,55 @@ bool Plane::parachute_manual_release()
>  
>      return true;
>  }
> +
> +// Code to detect loss of control for ArduPlane
> +#define PARACHUTE_CHECK_TRIGGER_SEC         1  // 1 second of loss of control triggers the parachute

You're right. I was just replicating what was in _ArduCopter/crash_check.cpp_ (although it probably has a define in the middle of the code because of the `#if`).

---
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/diydrones/ardupilot/pull/3703/files#r56144970
----==_mimepart_56e7e9549f2f3_5c9a3fba4a0e12a01564035
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

<p>In <a href="https://github.com/diydrones/ardupilot/pull/3703#discussion_r56144970">ArduPlane/parachute.cpp</a>:</p>
<pre style='color:#555'>&gt; @@ -56,3 +59,55 @@ bool Plane::parachute_manual_release()
&gt;  
&gt;      return true;
&gt;  }
&gt; +
&gt; +// Code to detect loss of control for ArduPlane
&gt; +#define PARACHUTE_CHECK_TRIGGER_SEC         1  // 1 second of loss of control triggers the parachute
</pre>
<p>You're right. I was just replicating what was in <em>ArduCopter/crash_check.cpp</em> (although it probably has a define in the middle of the code because of the <code>#if</code>).</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br>You are receiving this because you are subscribed to this thread.<br>Reply to this email directly or view it on GitHub:<br><a href="https://github.com/diydrones/ardupilot/pull/3703/files#r56144970">https://github.com/diydrones/ardupilot/pull/3703/files#r56144970</a><img alt="" height="1" src="https://github.com/notifications/beacon/AA3w3AXrej3tRwU19qTrurPwOFXgVo4xks5pto9UgaJpZM4HouR4.gif" width="1" /></p>
<div itemscope itemtype="http://schema.org/EmailMessage">
<div itemprop="action" itemscope itemtype="http://schema.org/ViewAction">
  <link itemprop="url" href="https://github.com/diydrones/ardupilot/pull/3703/files#r56144970"></link>
  <meta itemprop="name" content="View Pull Request"></meta>
</div>
<meta itemprop="description" content="View this Pull Request on GitHub"></meta>
</div>

----==_mimepart_56e7e9549f2f3_5c9a3fba4a0e12a01564035--